project-proposals/stage-2/stage-2-questionnaire.md

```diff
@@ -36,6 +36,7 @@
 ### What is the business roadmap of the project?
 
 *Please provide a link to the roadmap of the project.*
+*You can find an example here: https://osrd.fr/en/about/roadmap/*
```

Comment:
On the roadmap page it would be nice to have a link to the more detailed planning, to the issues or the projects in the GitHub projects.

Reply:
Done in: https://github.com/OpenRailAssociation/osrd-website/pull/263